packages/block-library/src/image/edit.native.js

```diff
@@ -382,86 +383,88 @@ export class ImageEdit extends React.Component {
 		};
 
 		const imageContainerHeight = Dimensions.get( 'window' ).width / IMAGE_ASPECT_RATIO;
-
 		const getImageComponent = ( openMediaOptions, getMediaOptions ) => (
-			<TouchableWithoutFeedback
```

Comment:
This change looks more dramatic than it actually is. All I'm doing here is wrapping `TouchableWithoutFeedback` in `Badgeable`